data/provider/src/main/java/tech/pegasys/teku/api/GetSpecResponse.java

```diff
@@ -82,12 +88,28 @@ private String getTargetAggregatorsPerCommittee() {
     return Integer.toString(ValidatorConstants.TARGET_AGGREGATORS_PER_COMMITTEE);
   }
 
+  private String getMaxRequestBlocks() {
+    return Integer.toString(specConfig.getNetworkingConfig().getMaxRequestBlocks().intValue());
+  }
+
   private String getRandomSubnetsPerValidator() {
-    return Integer.toString(ValidatorConstants.RANDOM_SUBNETS_PER_VALIDATOR);
+    return Integer.toString(specConfig.getNetworkingConfig().getSubnetsPerNode());
   }
 
   private String getEpochsPerRandomSubnetSubscription() {
-    return Integer.toString(ValidatorConstants.EPOCHS_PER_RANDOM_SUBNET_SUBSCRIPTION);
+    return Integer.toString(specConfig.getNetworkingConfig().getEpochsPerSubnetSubscription());
+  }
+
+  private String getAttestationSubnetCount() {
+    return Integer.toString(specConfig.getNetworkingConfig().getAttestationSubnetCount());
+  }
```

Comment:
if we're going to do this for all the others, we may as well change the gossip_max_size and max_chunk_size to be consistent...